package.json

```diff
@@ -6,7 +6,7 @@
     "coverage:collect": "ts-node ./scripts/collect-test-coverage.ts",
     "addwords": "node ./scripts/add-to-dict.js",
     "test-changed": "nx affected --target=test",
-    "test": "nx run-many --target=test --all",
+    "test": "nx run-many --target=test --all -- -i",
```

Comment:
nit: I prefer the verbose flag because it's easier to search

Is [--runInBand](https://jestjs.io/docs/cli#--runinband) correct?